examples/channel_inversion/inverse_problem.py

```diff
@@ -27,22 +27,24 @@
 do_taylor_test = not args.no_taylor_test
 no_exports = os.getenv('THETIS_REGRESSION_TEST') is not None
 
-# Create the solver object
+# Set output directory
 pwd = os.path.abspath(os.path.dirname(__file__))
+output_dir = f'{pwd}/outputs_new_' + '-'.join(controls) + '-opt'
+
+# Create the solver object
+# NOTE We disable all exports here as the they would only be
```

Comment:
I guess we should do the same for the Tohoku example as well then?

Reply:
Yeah, good point. Looking into it there seem to be a bunch of examples that need an updatel.